nginx/nginx.conf

```diff
@@ -0,0 +1,7 @@
+server { 
+  location / {
+    root /usr/share/nginx/html;
+    add_header Cross-Origin-Opener-Policy same-origin;
+    add_header Cross-Origin-Embedder-Policy require-corp;
```

Comment:
These security policies require us to host the images on the site from our origin, hence the direct serving of our images